frontend/packages/kubevirt-plugin/src/components/vms/virtualization.tsx

```diff
@@ -1,13 +1,13 @@
 import * as React from 'react';
 import { Helmet } from 'react-helmet';
-import { withStartGuide } from '@console/internal/components/start-guide';
+import { compose } from 'redux';
+import { connectToFlags, FlagsObject } from '@console/internal/reducers/features';
+import { FLAGS } from '@console/shared';
 import { HorizontalNav } from '@console/internal/components/utils';
+import { withStartGuide } from '@console/internal/components/start-guide';
```

Comment:
I think it should be:
import { withStartGuide } from '@console/internal/components/start-guide';
import { HorizontalNav } from '@console/internal/components/utils';
import { connectToFlags, FlagsObject } from '@console/internal/reducers/features';
import { FLAGS } from '@console/shared';

Reply:
hmm... ok :-)